README.md

```diff
@@ -12,18 +12,22 @@ $ rails generate mongoid:migration <your_migration_name_here>
 
 Run migrations:
 ```console
-$ rails db:migrate
-$ rails db:migrate:down VERSION=
-$ rails db:migrate:up VERSION=
-$ rails db:rollback
-$ rails db:rollback_to VERSION=
-$ rails db:migrate:redo
-$ rails db:migrate:reset
-$ rails db:migrate:status
-$ rails db:reseed (handled by mongoid)
-$ rails db:version
+$ rails db:mongoid:migrate
+$ rails db:mongoid:migrate:down VERSION=
+$ rails db:mongoid:migrate:up VERSION=
+$ rails db:mongoid:rollback
+$ rails db:mongoid:rollback_to VERSION=
+$ rails db:mongoid:migrate:redo
+$ rails db:mongoid:migrate:reset
+$ rails db:mongoid:migrate:status
+$ rails db:mongoid:reseed (handled by mongoid)
```

Comment:
This `reseed` task actually does not even exists in current versions of Mongoid, and it's not from this gem, so I think we can remove it.